db/db_impl/db_impl_readonly.cc

```diff
@@ -48,13 +48,14 @@ Status DBImplReadOnly::Get(const ReadOptions& read_options,
   SequenceNumber max_covering_tombstone_seq = 0;
   LookupKey lkey(key, snapshot);
   PERF_TIMER_STOP(get_snapshot_time);
-  if (super_version->mem->Get(lkey, pinnable_val->GetSelf(), &s, &merge_context,
-                              &max_covering_tombstone_seq, read_options)) {
+  if (super_version->mem->Get(lkey, pinnable_val->GetSelf(), nullptr, &s,
```

Comment:
Nit: to improve readability, it is preferred to add `/*timestamp=*/nullptr`.

Reply:
Added.